sql/test/insert_select_id.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE t1(i INTEGER, t TEXT, v INTEGER);
+
+INSERT INTO t1 VALUES (1, 'uid', 1);
+INSERT INTO t1 VALUES (local_sequence_number(), id(), 2);
```

Comment:
I assume we have somewhere where we do an insert like this but using a named key for e.g. id and a select with that name within a txn?

Reply:
Like https://github.com/SkipLabs/skdb/blob/main/sql/test/unit/test_id.sql, https://github.com/SkipLabs/skdb/blob/main/sql/test/unit/test_seqnum.sql, and https://github.com/SkipLabs/skdb/blob/main/sql/test/unit/test_seqnum2.sql, or something more?

Reply:
Yep, perfect.